runtime/drivers/duckdb/model_executor_objectstore_self.go

```diff
@@ -14,20 +14,27 @@ import (
 	"github.com/rilldata/rill/runtime/drivers/gcs"
 	"github.com/rilldata/rill/runtime/drivers/s3"
 	"github.com/rilldata/rill/runtime/pkg/fileutil"
+	"github.com/rilldata/rill/runtime/pkg/globutil"
 )
 
 var errGCSUsesNativeCreds = errors.New("GCS uses native credentials")
 
 type objectStoreInputProps struct {
-	Path   string             `mapstructure:"path"`
+	Path string `mapstructure:"path"`
+	// URI is renamed to path now. It is kept for backward compatibility.
+	URI    string             `mapstructure:"uri"`
 	Format drivers.FileFormat `mapstructure:"format"`
 	DuckDB map[string]any     `mapstructure:"duckdb"`
 }
 
 func (p *objectStoreInputProps) Validate() error {
-	if p.Path == "" {
+	if p.Path == "" && p.URI == "" {
 		return fmt.Errorf("missing property `path`")
 	}
+	if p.Path != "" && p.URI != "" {
+		return fmt.Errorf("cannot specify both `path` and `uri`")
+	}
+	p.Path = p.URI
```

Comment:
If `Path` is set and `URI` is not set, won't this override it? Shouldn't it be:

```suggestion
	if p.URI != "" { // Backwards compatibility
		p.Path = p.URI
	}
```